src/lib/utils/escape-highlight.ts

```diff
@@ -1,6 +1,6 @@
-import escape from '../lib/utils/escape';
-import isPlainObject from '../lib/utils/isPlainObject';
-import { Hit, FacetHit, EscapedHits } from '../types';
+import escape from './escape';
+import isPlainObject from './isPlainObject';
+import { Hit, FacetHit, EscapedHits } from '../../types';
```

Comment:
does this file have tests? if so, they should also be moved 👍

Reply:
I've moved it in the `utils/__tests__/` folder in https://github.com/algolia/instantsearch.js/pull/4618/commits/25110df6511569ee41bc6bae0d25087a68786efa